O/open62541/build_tarballs.jl

```diff
@@ -44,7 +44,8 @@ products = [
 ]
 
 # Dependencies that must be installed before this package can be built
-dependencies = Dependency[
+dependencies = [
+    Dependency("OpenSSL_jll"; compat="3.0.13")
```

Comment:
Please keep an older version for wider compatibility.
```suggestion
    Dependency("OpenSSL_jll"; compat="3.0.8")
```